src/Microsoft.Diagnostics.EventSource.Redist/dir.props

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <PackageVersion>2.0.0</PackageVersion>
+    <AssemblyVersion>2.0.0.0</AssemblyVersion>
+    <AssemblyKey>MSFT</AssemblyKey>
```

Comment:
Do we have to use the microsoft key for this guy?  I bet the open key is more inline with its use cases, but that'd be a breaking change over previous versions.

Reply:
It is likely to be a breaking change.  Is there guidance on what to use each key for?

Reply:
The open key is handy because it's not owned by Microsoft.  The private key is available in the open.  Anyone can build a copy of the assembly and have it be strong-name-valid.  This is what we use for all new assemblies.  For old assemblies which have to maintain binary compatibility with some previous version we use the old key.

Reply:
We don't have a compatibility requirement here, however most users ARE Microsoft people incorporating it into products they ship.   I am assuming the new key is still fine.  If so we should do that.

Reply:
I've switched to the open key.  If we find that we need the Microsoft key, we can easily switch back.